app/assets/stylesheets/application.scss

```diff
@@ -20,6 +21,7 @@
 @import "govuk_publishing_components/components/phase-banner";
 @import "govuk_publishing_components/components/radio";
 @import "govuk_publishing_components/components/search";
+@import "govuk_publishing_components/components/skip-link";
```

Comment:
Looks like we use skip-link on line 30 of `application.html.erb` so :+1:

Reply:
Yeah, not sure how I missed this before!